### PR TITLE
Appium runner improvements

### PR DESCRIPTION
**Pull request by alsorokin**
* Changed ios device in appium config to match the emulator on osx slave
* Added 'common' folder to the search paths for Appium runner
* Save appium log
* Pass helpers to tests
* Untied from mobilespec
* Extended wdHelper with getWebviewContext() function
* Added InjectLibraries function to wdHelper

### CODE REVIEW

File: lib/appium/helpers/wdHelper.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    return driver
+        .init(driverConfig)
+        .setImplicitWaitTimeout(IMPLICIT_WAIT_TIMEOUT);
+};
+
+module.exports.getWD = function () {
+    return global.WD;
+};
+
+module.exports.getWebviewContext = function (driver) {
```
Comment (vladimir-kotikov) [inline lib/appium/helpers/wdHelper.js]: What if the webview couldn't be loaded somehow? Maybe we need to limit number of attempts? 
```js
getWebviewContext = function (driver, maxAttempts) {
...
    return getWebviewContext(driver, maxAttempts - 1);
});
```

File: lib/appium/helpers/wdHelper.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+var APPIUM_SERVER_HOST = 'localhost';
+var APPIUM_SERVER_PORT = 4723;
+var WEBVIEW_WAIT_TIMEOUT = 5000;
+var IMPLICIT_WAIT_TIMEOUT = 10000;
+
+var fs = require('fs');
+var path = require('path');
+
+module.exports.getDriver = function (platform) {
```
Comment (vladimir-kotikov) [inline lib/appium/helpers/wdHelper.js]: Is the driver instance supposed to be a singleton?

File: medic/medic-appium.js [hunk 3]
```diff
@@ -197,9 +214,10 @@ function parseArgs() {
     return options;
 }
 
-function getLocalCLI() {
-    if (util.isWindows()) {
-        return "cordova.bat";
+function getLocalCLI(appPath) {
+    if (util.isWindows() || !fs.existsSync(path.join(getFullAppPath(appPath), "./cordova"))) {
```
Comment (vladimir-kotikov) [inline medic/medic-appium.js]: Why we fall back to default on Windows? I can see `cordova.bat` in mobilespec directory on Windows

Comment (alsorokin) [inline medic/medic-appium.js]: Windows doesn't need the dot-slash to execute this .bat
So basically, on windows "cordova" command would execute .bat and if it's not there, will execute global cordova.

Comment (vladimir-kotikov) [inline medic/medic-appium.js]: Right, thanks for clarification.

File: buildbot-conf/cordova-config.json.sample [hunk 4]
```diff
@@ -12,8 +12,8 @@
     "appium": {
         "androidDeviceName":      "Galaxy_Nexus",
         "androidPlatformVersion": "19",
-        "iosDeviceName":          "iPhone 5",
-        "iosPlatformVersion":     "8.4",
+        "iosDeviceName":          "iPhone 6s",
+        "iosPlatformVersion":     "9.1",
         "timeout":                2400,
```
Comment (dblotsky) [inline buildbot-conf/cordova-config.json.sample]: Just a friendly reminder to make this change in the actual file on the Buildbot server; this file is just a sample.

Comment (alsorokin) [inline buildbot-conf/cordova-config.json.sample]: Thanks for the reminder, it's already there.

**Pull request merged**